Doc/library/asyncio-task.rst

```diff
@@ -631,7 +631,7 @@ Shielding From Cancellation
 Timeouts
 ========
 
-.. coroutinefunction:: timeout(delay)
+.. function:: timeout(delay)
 
     An :ref:`asynchronous context manager <async-context-managers>`
```

Comment:
Maybe
```suggestion
    Returns an :ref:`asynchronous context manager <async-context-managers>`
```
since `timeout` itself technically isn't a context manager.

Reply:
Applied, but in the imperative mood for consistency with the rest of the module